cmd/fyne/internal/commands/package-unix.go

```diff
@@ -96,8 +97,14 @@ func (p *Packager) packageUNIX() error {
 			return fmt.Errorf("failed to write Makefile string: %w", err)
 		}
 
+		tarCmdArgs := []string{"-Jcf", filepath.Join(p.dir, p.Name+".tar.xz")}
+		if runtime.GOOS == "openbsd" {
```

Comment:
if OpenBSD cannot handle writing tar files with .xz compression should we not also ensure that we only produce .gz files when it is the /target/ system as well? Otherwise won't the user be unable to open the package if it was produced on, say, a Linux host?

Reply:
Good point with regards to logic and consistency, even though practically I haven't seen anyone targeting OpenBSD in a cross build from another system. It seems impractical to achieve and maintain as the ABI changes pretty much every six months with a new release.
Still, it should depend on the target instead of the runtime OS, and I'll adjust it.

Reply:
Better now, thanks.